include/worker.hpp

```diff
@@ -110,6 +110,16 @@ class Worker
         return m_isAllFruCollected;
     }
 
+    /**
+     * @brief API to get system config JSON object
+     *
+     * @return System config JSON object.
+     */
+    inline nlohmann::json getSysConfigJsonObj() const
```

Comment:
why getSysConfigJsonObj and not getSysCfgJsonObj

Reply:
no harm in expanding a word in function name. it gives better readability only.
we discussed to shorten the names only for variables as variables will be used in multiple lines within the same api which increases the number of lines.

Reply:
done